### PR TITLE
Create lab-reservation-policy.md

### PR DESCRIPTION
**Pull request by edricx**
Transcription of lab reservation policy written by Alex Welty

@sahilhasan

### CODE REVIEW

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[[!meta title="Lab Reservation Policy"]]
+
```
Comment (kpengboy) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: Should add an annotation here describing who established the policy (the Board of Directors, in this case) and when. This lets us know who is allowed to change the policy in the future.

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+[[!meta title="Lab Reservation Policy"]]
```
Comment (kpengboy) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: By convention we only uppercase the first letter of the first word, e.g. "Lab reservation policy"

**Comment by abizer** [comment]
The document this was based on has been updated and had several comments resolved, could you please bring this version up to date with the one in the Drive folder?

**Comment by sahilhasan** [comment]
@abizer this document is consistent with the document voted on by BoD, no text changes (besides the casing issues @kpengboy mentioned) are needed afaik

**Comment by kpengboy** [comment]
Also he missed some finer changes like the part about "all regular OCF policies are still in effect"

**Comment by abizer** [comment]
This document looks quite different from the working copy we have in Google Docs though. How did BoD vote on the old copy when we had already merged in many of the comments prior to BoD? Among many other things for example this document says things like the reservation will be approved by OCF staff, instead of the OCF BoD, etc.

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+The Board of Directors holds the authority to approve requests to reserve the
+OCF. When the OCF is not formally in session (i.e. during breaks), the Cabinet
+can make provisional decisions on requests. If the decisions are ratified at
+the first Board meeting of the next formal session, they will become valid;
+otherwise they will expire.
+
+## Cancellation and rescheduling
+
+The OCF is not responsible for events outside of our control that can interfere with scheduled events. These can include, but are not limited to: fire alarms, building closures, etc. In situations where events are interrupted, the OCF may be willing to attempt to reschedule the event, but cannot guarantee that an event will be rescheduled. In addition, failure to abide by safety procedures (e.g not evacuating during a fire alarm) may result in student groups being banned from reserving the OCF, banning accounts affiliated with the event as well as possible further action as decided by the Board of Directors.
```
Comment (matthew-mcallister) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: Linewrapping yo

**Comment by sahilhasan** [comment]
@abizer Interesting...I hadn't noticed that. @edricx looks like you copied over the wrong one entirely

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+outside population (excluding OCF front-desk staff) is readily available.
+
+Here are a few important things to note:
+
+- All OCF computers run on Debian Linux, so operating system specific
+  applications such as Microsoft Word will not be available. However,
+  browser-based tools, such as Google Docs, will work.
+
+- The OCF runs a decal from 8 to 9 pm on Tuesdays and Thursdays, and as such we
+  will be unable to allow lab reservations within that timeframe.
```
Comment (matthew-mcallister) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: What about BoD and staff meetings on Mondays?

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+[[!meta title="Lab reservation policy"]]
+
+Registered Student Organizations of UC Berkeley are able to reserve the OCF lab
+space for a maximum of 2 hours at a time during the OCF’s regular operating
+hours. Those who reserve the OCF lab space are able to access the 28 computers
+within the lab (as well as additional seating), in addition to the main TV
+attached to the back wall. The OCF can additionally provide a whiteboard and
+printing (pursuant to printing quotas) to student groups who utilize the lab
+space. Should the lab be reserved, the option to fully close the lab to the
+outside population (excluding OCF front-desk staff) is readily available.
```
Comment (matthew-mcallister) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: "readily" available? Why not "available if necessary"?

**Comment by abizer** [comment]
@matthew-mcallister https://drive.google.com/open?id=1xq1a7-EsMYRloOzcQgn6x88PbFCN1ILwHc7UtLEqoaA

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md (hunk 5 above)
Comment (abizer) [reply ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: I think we covered most of these things in the comments on the Drive doc

Comment (sahilhasan) [reply ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: That was changed. I would hold off on reviewing wording until @edricx puts the new one in :)

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md (hunk 4 above)
Comment (sahilhasan) [reply ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: We may be willing to cancel Staff or BoD on any particular week (as we are doing for the FB event next week). Not saying we will say yes, but it is not unheard of.

**Comment by edricx** [comment]
@sahilhasan I didn't close Alex's original doc yesterday, probably clicked the wrong tab and copied from his doc. Fixing rn

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+then be approved by  the OCF Board of Directors.  Should the request be
+approved, the student organization will be allowed to to use the lab at the
+allotted time pursuant to the following conditions:
+
+1. The lab space must be cleaned after use by the student organization. Eating
+and drinking in the lab is permitted, as long as the space remains clean and
+available for use immediately after the event concludes.
+
+2. Printing in the OCF lab is subject to the standard per person rules located
+here. Please note that the OCF currently does not offer printing for group
```
Comment (abizer) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: located here needs a link

Comment (edricx) [reply ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: Missed the hyperlinks, adding now.

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+event will be rescheduled. In addition, failure to abide by safety procedures
+(e.g not evacuating during a fire alarm) may result in student groups being
+banned from reserving the OCF, banning accounts affiliated with the event as
+well as possible further action as decided by the Board of Directors.
+
+## Liability
+
+The OCF is not liable for the actions taken by those who attend events within
+the lab, and will hold the hosting group liable for any policy violations. In
+addition, the OCF is not liable for any damages during said event.
```
Comment (abizer) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: this paragraph could stand to be worded a bit more clearly

File: ocfweb/docs/docs/services/lab/lab-reservation-policy.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+## Amendment of this policy
+
+The Cabinet may update purely factual information in this policy (e.g. the
+dates and times of the decal); all other amendments may only be made by the
```
Comment (abizer) [inline ocfweb/docs/docs/services/lab/lab-reservation-policy.md]: DeCal

**Comment by abizer** [comment]
There is some odd spacing as well, like repeated spaces and whatnot that should be corrected prior to merging

**Comment by kpengboy** [comment]
This is not quite done yet, it's still missing the enactment annotation.

**Comment by kpengboy** [comment]
Also, I forgot, can you add links to the two policies from our [Policies list](https://www.ocf.berkeley.edu/docs/docs/policies/) and [Staff policies list](https://www.ocf.berkeley.edu/docs/staff/policies/)?

**Pull request merged**